src/Kiota.Builder/Extensions/StringExtensions.cs

```diff
@@ -94,5 +94,20 @@ public static string ReplaceDotsWithSlashInNamespaces(this string namespaced)
             var parts = namespaced.Split('.');
             return string.Join('\\', parts.Select(x => x.ToFirstCharacterUpperCase())).Trim('\\');
         }
+        private static readonly Regex propertyCleanupRegex = new(@"[""\s!#$%&'()*+,./:;<=>?@\[\]\\^`{}|~]", RegexOptions.Compiled);
```

Comment:
Can you add a comment that describes which characters this regex matches? There might be a missing escape for the forward slash.

```suggestion
        private static readonly Regex propertyCleanupRegex = new(@"[""\s!#$%&'()*+,.\/:;<=>?@\[\]\\^`{}|~]", RegexOptions.Compiled);
```

Reply:
+1

Reply:
the forward slash is already present, right before the ^ symbol (and twice because it needs to be escaped). Added a comment.

Reply:
May the mysteries of regex be revealed to me someday.